pythonfile/flask_api.py

```diff
@@ -36,29 +39,103 @@ def add_project():
 
     if error == None:
         manifest['id']=uuid.uuid4()
+        #inserts in mongo db
         coll.insert(manifest)
-        elastic.store_json("test", "projects", manifest)
+        #inserts in elasticsearch now
+        res = es.index(index="projects-index", doc_type='Project', id=manifest['id'], body=manifest)
         return make_response(manifest['id'])
     else:
         return make_response((error, '302'))
 
-@app.route('/api/projects/<uuid:project_id>', methods=['GET'])
+@app.route('/api/projects/get/<project_id>')
 def get_project_by_id(project_id):
     res=coll.find_one({'_id':project_id})
     return jsonify(res)
 
-@app.route('/api/projects/<uuid:project_id>', methods=['DELETE'])
+
+@app.route('/api/projects/delete/<project_id>')
 def delete_project(project_id):
-    if coll.delete_one({'_id':project_id}).deleted_count != 0:
+    #deletion from elastic search index
+    try:
+        res = es.delete(index="projects-index", doc_type='Project', id=project_id, refresh=True)
+    except NotFoundError:
+        return make_response('Project not found', 404)
+    except:
+        return make_response('Unexpected Error', 500)
+    #Deletion from Mongodb and return
+    if (coll.delete_one({'_id':project_id}).deleted_count != 0):
         return make_response('Success')
     else:
         return make_response('Project not found', 404)
 
-# receive body of elasticsearch query
+
+
+#Note, works but need better exception handlin on empty index
 @app.route('/api/projects/search', methods=['POST'])
+#@app.route('/api/projects/search')
 def search():
-    res=es.search(index="test", doc_type="projects", body=request.json)
-    return jsonify(res)
+    try:
+        res=es.search(index="projects-index", doc_type="Project", body=request.json)
+        return jsonify(res)
+    except RequestError:
+        return ("Some Error")
+    except:
+       return ("Error: Index probablly empty!")
+
+# dummy add a few projects to es
+#Note: deleting these projects will throw not found error bcause they are in es only, but they are deleted
+@app.route('/api/projects/dummyadd', methods=['GET'])
```

Comment:
This is nice for testing, but:
1. We already have example files in the repository
2. This will be unnecessary once the POST /api/projects call is working.

So while it is nice, that you got yourself a solution for testing, please remove (all of) this for merge.
I do not want debug code in my master. Even if we decide to use this for a demo, this can live in a separate branch.